assets/Icons/raw/skype.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
need to remove:
* xml tag
* comments
* title, desc, defs tags
* ids from g tags (4 of them)

Reply:
wait this is in the `raw` folder. why do we have icons in `Assets` and `components/Icon/raw` ?

Reply:
As a reference in case we need to re-optimize the SVG.

Reply:
I haven't received the 240v240 icons yet. Hopefully we can develop a process where UX keeps the assets/Icons folder updated. I'll talk to him today.

There is no reason why we can't go out with the currently-sized icons which are the ones we have right now.

Reply:
If the raw folder causes confusion, let's get rid of it. I have found it important to be able to reference the original SVG, however.

Reply:
is "raw" the same as what Design provides in the Assets directory?

Reply:
The assets/Icons folder did not exist before. I gave that folder the name `raw`. Manuel can choose another label if he'd like. I'll talk to him soon.

Reply:
ok i see now. the optimized version should be a temporary "build" step once we have this task automated, but let's push this out and worry about that later